packages/tempus/src/Tempus_StepperSubcycling_impl.hpp

```diff
@@ -98,7 +104,32 @@ StepperSubcycling<Scalar>::StepperSubcycling(
     this->initialize();
   }
 }
+#endif
+
+template<class Scalar>
+StepperSubcycling<Scalar>::StepperSubcycling(
+  const Teuchos::RCP<const Thyra::ModelEvaluator<Scalar> >& appModel,
+  const Teuchos::RCP<IntegratorBasic<Scalar> >& scIntegrator,
+  bool useFSAL,
+  std::string ICConsistency,
+  bool ICConsistencyCheck,
+  const Teuchos::RCP<StepperSubcyclingAppAction<Scalar> >& stepperSCAppAction)
+{
+  this->setStepperType(        "Subcycling");
+  this->setUseFSAL(            useFSAL);
+  this->setICConsistency(      ICConsistency);
+  this->setICConsistencyCheck( ICConsistencyCheck);
 
+  this->setObserver();
```

Comment:
Need an ifdef Tempus_HIDE_DEPRECATED_CODE around this observer call.

Reply:
Made this change.

Reply:
I also added the AppAction unit tests.